### PR TITLE
Add a change callback

### PR DESCRIPTION
**Pull request by ahilsend**

### CODE REVIEW

File: dist/directives/date-picker.js [hunk 1]
```diff
@@ -162,6 +161,18 @@ function datePicker($log, $compile, $filter, $document, $timeout) {
...
+        function updateModal(value) {
+
+            scope.model = value.clone();
+            ngModelController.$setDirty();
+
+            $timeout(function () {
+                scope.onChangeCallback({
+                    'model': scope.model
+                });
+            }, 0, false);
```
Comment (artgryn) [inline dist/directives/date-picker.js]: what the idea of this?

Comment (ahilsend) [inline dist/directives/date-picker.js]: to know when the value changed without having to add annoying watchers everywhere

File: dist/directives/date-picker.js [hunk 2]
```diff
@@ -162,6 +161,18 @@ function datePicker($log, $compile, $filter, $document, $timeout) {
             return scope.timezone || moment.tz.guess();
         }
 
+        function updateModal(value) {
```
Comment (nicolas-nannoni) [inline dist/directives/date-picker.js]: You mean `model`?

File: dist/directives/date-picker.js (hunk 1 above)
Comment (artgryn) [inline dist/directives/date-picker.js]: I can not find where the `cancel` for timeout

**Comment by erikpalsson** [comment]
This should be all good stuff! :+1: 

I got confused though because I recognized a few changes (not related to the callback) from the last pull request here by Marko that was towards the source folder. Compare these versions and they are slightly different:
https://github.com/Accedo-Products/angular-moment-datetime/blob/master/dist/directives/date-picker.js
https://github.com/Accedo-Products/angular-moment-datetime/blob/master/source/directives/date-picker.js

File: dist/directives/date-picker.js (hunk 1 above)
Comment (ahilsend) [inline dist/directives/date-picker.js]: well, it should run right after the current digest, thus the 0. And should not be cancelled. This is the same pattern we use in the dropddown-directives we have

File: dist/directives/date-picker.js (hunk 2 above)
Comment (ahilsend) [inline dist/directives/date-picker.js]: hm, yeah ^^

**Comment by ahilsend** [comment]
@erikpalsson correct. I guess the previous PR didn't include the run of 'npm run build', which will update the 'dist' files

**Comment by gouegd** [comment]
@ahilsend ah good point, if that happens again you may consider using ghooks (npm hooks on git actions), see https://www.npmjs.com/package/ghooks

**Pull request merged**